libcxx/test/libcxx/ranges/range.adaptors/range.adaptor.helpers/as-lvalue.pass.cpp

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03
```

Comment:
I'd prefer if we did `UNSUPPORTED: c++03, c++11` so we can use C++14 constexpr. It's nice that you try to make it work in C++11, but TBH I think we should prioritize readability for these tests -- at the end of the day `__as_lvalue` is also not going to be used in C++11 mode so I'm not too shocked about not having coverage for it in that mode.

Then you can use multiple statements in the function below.